### ISSUE TITLE
Can't download files - silent JavaScript error when clicking "download"

### ISSUE THREAD

**Issue by ethankaminski** [itchio/itch.io#683]
I can't currently download game releases from the itch.io website. When I click on the "Download" link, it generates a JS error (not displayed in the UI, only visible in the browser's dev console) about incorrectly formatted JSON.

Looks like it's trying to JSON.parse something which is URI encoded - the string is `%5b"2%3a4%3a1%3a159664"%5d` (checked by monkey patching JSON.parse to intercept and log that).

The stack trace for the error I'm seeing:

```
VM515:1 Uncaught SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse [as _parse] (<anonymous>)
    at JSON.parse (<anonymous>:1:62)
    at Function.I.ConversionTracker.ConversionTracker.get_cookie (bundle.min.js?1500422271:3)
    at Function.I.ConversionTracker.ConversionTracker.find_click (bundle.min.js?1500422271:3)
    at Function.I.ConversionTracker.ConversionTracker.after_click_action (bundle.min.js?1500422271:3)
    at Function.I.ConversionTracker.ConversionTracker.download (bundle.min.js?1500422271:3)
    at ViewGame.el.dispatch.download_btn (game.min.js?1500422271:1)
    at HTMLDivElement.<anonymous> (bundle.min.js?1500422271:1)
    at HTMLDivElement.dispatch (lib.js?1500422271:2)
    at HTMLDivElement.y.handle (lib.js?1500422271:2)
```

Using Chrome on Linux with a logged in user account.

**Comment by fasterthanlime** [comment]
Can you post the URL of the game page you were trying to download?

(and your itch.io username? )

**Comment by ethankaminski** [comment]
One game where I had the issue was https://bolchegl.itch.io/foxrush. Username is "Soron".

**Comment by fasterthanlime** [comment]
Thanks!

> One game where I had the issue

Does it happen every time you go to that game page and try to download? Or did it just happen once?

**Comment by ethankaminski** [comment]
It occurs 100% consistently for me so far, including on other game pages from this same jam. I've just checked a few more, and all 5 game pages that I've tested it with show the same bug preventing downloads (both on the general game page, and the jam-specific page).

Trying a random non-jam game (https://joyato.itch.io/one-week-my-room), the same issue occurs: unable to download, due to a JSON.parse error.

Browser-embedded games work fine (Unity, PICO-8, custom HTML5 that pops up in a new window, etc.). Similarly, a name-your-price game popped up a modal window, as expected (I did not investigate further at that point).
